roles/satellite-clone/tasks/pre_install_check.yml

```diff
@@ -1,4 +1,6 @@
 ---
+- include: pre_install_check_migration.yml
+  when: satellite_migration
```

Comment:
nitpick but we can probably put a space after this since its unrelated to the task below